### DISCUSSION TITLE
SH-wg

### DISCUSSION THREAD

**Discussion by segabrielssj** [hatlabs/discussions#48] [General]
Will the SH-wg device also present AIS data over wifi?

**Comment by mairas** [comment]
Yes it does, as long as the data is available on the N2K network!

**Comment by segabrielssj** [comment]
Thanks for the update.
Will consider to order one soon then 😉.

Mvh
Johan Gabrielsson
+46 733 363060

On 10 Apr 2023, at 11:14, Matti Airas ***@***.***> wrote:

﻿

Yes it does, as long as the data is available on the N2K network!

**Comment by MikeP-NZ** [comment]


**Reply by mairas** [reply]
Hi Mike,

Thank you for the compliments! Even though the SH-wg firmware is open source and therefore adapting it to different hardware is fair game, I would strongly recommend purchasing the actual product instead. The pricing is still extremely competitive compared to other similar products and the product sales are my only means of recouping (and justifying) the considerable amount of time and effort I invested in developing the firmware. :-)

**Comment by robdekker2** [comment]
Maybe a stupid question but I have my sh-wg Connected to the same network as my raspberry but signal k comes with the following fault: getaddrinfo ENOTFOUND sh-wg. Reconnect sh-wg 2223 retry... What am I doing wrong?

**Reply by mairas** [reply]
Documenting the solution for @robdekker2's problem for posterity: the hostname to connect depends a bit on the DNS settings, but generally `sh-wg.local`, the standard domain for mDNS resolution (in which the hostname is found using a multicast in the local network instead of asking a DNS server) works more reliably than just bare `sh-wg`. Some routers also provide DNS hostnames for DHCP clients. In such cases, hostnames such as `sh-wg.lan` would also work.
